.dockerignore

```diff
@@ -1,3 +1,5 @@
 tmp/
```

Comment:
Minor, but can you alphabetize? Makes it easier as file grows

Reply:
sure, done.